db/models.py

```diff
@@ -1 +1,33 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(
+        "This field describes what kind of bonus players can get from it",
+        max_length=255
+    )
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(null=True)
+
+
+class Player(models.Model):
+    nickname = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(max_length=255)
+    bio = models.CharField(
+        "It stores a short description "
+        "provided by a user about himself/herself.",
```

Comment:
This parameter is being used as verbose_name. It's not the best choice for verbose name in your case. 
You can read about verbose names here https://www.geeksforgeeks.org/verbose_name-django-built-in-field-validation/
Or here: https://docs.djangoproject.com/en/4.2/ref/models/options/#verbose-name
Remove this documentation string everywhere, please.